XcodeServerSDK.xcodeproj/project.pbxproj

```diff
@@ -118,6 +118,12 @@
 		709ED67C1B35FD3F00A06038 /* XcodeServerEntityTests.swift in Sources */ = {isa = PBXBuildFile; fileRef = 709ED67B1B35FD3F00A06038 /* XcodeServerEntityTests.swift */; };
 		709ED6801B3608FC00A06038 /* XcodeServerConfigTests.swift in Sources */ = {isa = PBXBuildFile; fileRef = 709ED67F1B3608FC00A06038 /* XcodeServerConfigTests.swift */; };
 		70B21FF81B3AFB1D00EAD4EB /* BotConfigurationTests.swift in Sources */ = {isa = PBXBuildFile; fileRef = 70B21FF71B3AFB1D00EAD4EB /* BotConfigurationTests.swift */; };
+		70B43A861B43C95A004F9DEB /* XcodeServer+Device.swift in Sources */ = {isa = PBXBuildFile; fileRef = 70B43A851B43C95A004F9DEB /* XcodeServer+Device.swift */; };
+		70B43A871B43C95A004F9DEB /* XcodeServer+Device.swift in Sources */ = {isa = PBXBuildFile; fileRef = 70B43A851B43C95A004F9DEB /* XcodeServer+Device.swift */; };
+		70B43A881B43C95A004F9DEB /* XcodeServer+Device.swift in Sources */ = {isa = PBXBuildFile; fileRef = 70B43A851B43C95A004F9DEB /* XcodeServer+Device.swift */; };
```

Comment:
I've noticed that for each platform (OS X, iOS, watchOS) is created duplicate, so that's why @esttorhe  had to commit fix with missing `Repository`.